.gitignore

```diff
@@ -1,15 +1,9 @@
-build
-build/
-build/*
+build/**
+!build/.gdignore
 export_presets.cfg
-.import
 .import/
-.import/*
-.godot
 .godot/
-.godot/*
 *.translation
 .mono/
-addons/discord_game_sdk/bin
-addons/discord_game_sdk/bin/
-addons/discord_game_sdk/bin/*
+addons/discord_game_sdk/discord_game_sdk.*
+addons/discord_game_sdk/libdiscord_game_sdk.*
```

Comment:
I have not checked the documentation, but as far as I'm aware the library files need to go inside of bin, and this new pattern would not accommodate for that.

Reply:
The Game SDK is no longer loaded from the `bin` directory, at least on Windows. I tried moving them into `addons/discord_game_sdk/bin` and I got this error:
```
Can't open dynamic library: C:/Users/atiru/Projects/sound-space-plus/addons/discord_game_sdk/discord-game-sdk-godot.dll, error: Error 126: The specified module could not be found.
```

Reply:
It seems that bin is now unused